src/main.cpp

```diff
@@ -1,6 +1,29 @@
 #include <iostream>
+#include <algorithm>
+#include <random>
+
+#include "sort/sort.hpp"
+#include "terminal_ui/terminal_ui.hpp"
+
+#define ARR_SIZE 20
+
+std::vector<uint16_t> generate_array(){
+    std::vector<uint16_t> arr;
+    for(int i = 1; i <= ARR_SIZE; i++){
+        arr.push_back(i);
+    }
+
+    std::shuffle(arr.begin(), arr.end(), std::random_device());
+    return arr;
+}
 
 int main() {
-    std::cout << "Hello world!" << std::endl;
+
+    // Below code for testing.
+    std::vector<uint16_t> arr = generate_array();
+    //bubble_sort(arr, &display_array);
+    //selection_sort(arr, &display_array);
+    insertion_sort(arr, &display_array);
+    //merge_sort(arr, &display_array);
     return 0;
-}
+}
```

Comment:
This file ends with no newline right? You need to add newline to the end of each file. So it should end like this:
```suggestion
}

```
Check it in your editor (VSCode). If it already ended with a newline, then it's fine. If not then make sure this and all other files end with a newline. Based on best practice in Linux/Unix world, a file is not really a text file if it does not end with a newline.